chat/title-bar/src/TitleBar/TitleBar.tsx

```diff
@@ -45,7 +45,7 @@ export const TitleBar = forwardRef(
               [contentAlignmentStyles]: align === Align.Center,
             })}
           >
-            <Avatar variant="mongo" color="green-base" sizeOverride={24} />
+            <Avatar variant="mongo" sizeOverride={24} />
```

Comment:
nit: could we use `ChatAvatarVariant.Mongo`?

Reply:
Technically yeah. I was trying to change as little as possible to the `chat` components